docs/core/porting/libraries.md

```diff
@@ -49,19 +49,19 @@ A key thing to understand is that **a project targeting a lower version cannot r
 
 It's recommended that you pick the lowest possible .NET Standard version and use that throughout your project.
 
-Read more in [.NET Platform Standard Library](../standard/library.md).
+Read more in [.NET Platform Standard Library](../../standard/library.md).
 
 ## Key Technologies Not Yet Available on the .NET Standard or .NET Core
 
 You may be using some technologies available for the .NET Framework that are not currently available for .NET Core.  Each of the following sub-sections corresponds to one of those technologies.  Alternative options are listed if it is feasible for you to adopt them.
 
 ### App Domains
 
-AppDomains can be used for different purposes on the .NET Framework. For code isolation, we recommend separate processes and/or containers as an alternative. For dynamic loading of assemblies, we recommend the new [AssemblyLoadContext](http://dotnet.github.io/api/System.Runtime.Loader.AssemblyLoadContext.html) class.
+AppDomains can be used for different purposes on the .NET Framework. For code isolation, we recommend separate processes and/or containers as an alternative. For dynamic loading of assemblies, we recommend the new  @System.Runtime.Loader.AssemblyLoadContext class.
 
 ### Remoting
 
-For communication across processes, inter-process communication (IPC) mechanisms can be used as an alternative to Remoting, such as [Pipes](https://.docs.microsoft.com/en-us/dotnet/core/api/system.io.pipes) or [Memory Mapped Files](hhttps://docs.microsoft.com/en-us/dotnet/core/api/system.io.memorymappedfiles.memorymappedfile#System_IO_MemoryMappedFiles_MemoryMappedFile).
+For communication across processes, inter-process communication (IPC) mechanisms can be used as an alternative to Remoting, such as @System.IO.Pipes or @System.IO.MemoryMappedFiles.MemoryMappedFile.
```

Comment:
I can roll back the change for the second one and keep as an absolute URL (just fixing the extra we have in hhttp). There's no way to format the API name like we can't format for msdn.